cmd/flux/create_helmrelease.go

```diff
@@ -130,6 +131,7 @@ func init() {
 	createHelmReleaseCmd.Flags().StringVar(&helmReleaseArgs.saName, "service-account", "", "the name of the service account to impersonate when reconciling this HelmRelease")
 	createHelmReleaseCmd.Flags().StringArrayVar(&helmReleaseArgs.valuesFiles, "values", nil, "local path to values.yaml files")
 	createHelmReleaseCmd.Flags().Var(&helmReleaseArgs.valuesFrom, "values-from", helmReleaseArgs.valuesFrom.Description())
+	createHelmReleaseCmd.Flags().Var(&helmReleaseArgs.crds, "crds", helmReleaseArgs.crds.Description())
```

Comment:
```suggestion
	createHelmReleaseCmd.Flags().Var(&helmReleaseArgs.crds, "upgrade-crds", helmReleaseArgs.crds.Description())
```

maybe?

Reply:
I would leave it like it is, since we also set the install value.